src/components/TicketDetailModal/TicketDetailModal.tsx

```diff
@@ -0,0 +1,308 @@
+import {
+  AccessTime,
+  Add,
+  AddBox,
+  ArrowForward,
+  AttachFile,
+  Check,
+  Close,
+  CropLandscape,
+  FileCopy,
+  FormatListBulleted,
+  Inbox,
+  Label,
+  Notes,
+  Person,
+  Remove,
+  Share,
+  Style,
+  Title,
+  Visibility,
+  WorkOutline,
+} from "@material-ui/icons";
+import styled from "styled-components";
+
+import { GET_TICKET_BY_ID } from "../../api/tickets";
+import { ProjectContext } from "../../context/project";
+import { useFetch } from "../../hooks/useFetch";
+import { useTypeSafeContext } from "../../hooks/useTypeSafeContext";
+import TicketDefenition from "../../interfaces/Ticket";
+import Avatar from "../Avatar";
+import _Button from "../Button";
+import _Card from "../Card";
+import Dialog from "../Dialog";
+import EditableLabel from "../EditableLabel";
+import _EditableTicket from "../EditableTicket";
+import IconButton from "../IconButton";
+
+export interface TicketDetailModalProps {
+  isOpen: boolean;
+  ticketId: number | null;
+  onClose?: () => void;
+}
+
+export default function TicketDetailModal({ isOpen, ticketId, onClose }: TicketDetailModalProps): JSX.Element {
+  const project = useTypeSafeContext(ProjectContext);
+  const { data: ticketResponse } = useFetch<TicketDefenition>(GET_TICKET_BY_ID(project.id, ticketId!));
+
+  return (
+    <Dialog isOpen={isOpen} onClose={onClose} disableCloseButton>
+      <Card>
+        <Container>
+          <CloseIconButton onClick={onClose}>
+            <Close />
+          </CloseIconButton>
+
+          <Headline>
+            <Title />
+
```

Comment:
No offense, but when I see your JSX file I always feel the same as when I see teeth with gaps lol

![image](https://user-images.githubusercontent.com/44686790/172988672-19072ca4-2e86-445e-be75-2151b404566f.png)

Reply:
HTML (or jsx) is not only tool to build a web page but a document. And shorter or small chunk is always easy to understand for other developers.

Reply:
My point is that they are too spacious - I think you are over doing it.